packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsCustom.twig

```diff
@@ -1,23 +1,26 @@
-<Breadcrumbs>
-  <ol>
-    <li class="d-none d-tablet-flex">
-        <Link href="#rootUrl" color="primary" isUnderlined>Root</Link>
-    </li>
-    <li class="d-none d-tablet-flex">
-        <Icon name="chevron-right" />
-        <Link href="#categoryUrl" color="primary" isUnderlined>Category</Link>
-    </li>
-    <li class="d-tablet-none">
-        <Icon name="chevron-right" />
-        <Link href="#subcategoryUrl" color="primary" isUnderlined>Custom go back link</Link>
-    </li>
-    <li class="d-none d-tablet-flex">
-        <Icon name="chevron-right" />
-        <Link href="#subcategoryUrl" color="primary" isUnderlined>Subcategory</Link>
-    </li>
-    <li class="d-none d-tablet-flex">
-        <Icon name="chevron-right" />
-        <Link href="#currentUrl" color="secondary" aria-current="page">Current page</Link>
-    </li>
-  </ol>
-</Breadcrumbs>
+<section class="docs-Section">
+  <h2 class="docs-Heading">Custom</h2>
+    <Breadcrumbs>
+    <ol>
```

Comment:
There should be more space as this is inside the Breadcrumbs. Please fix the indentation.

Reply:
fixed